deploy/helm/elastic-agent/README.md

```diff
@@ -123,12 +123,21 @@ The chart built-in [kubernetes integration](https://docs.elastic.co/integrations
 | kubernetes.system.metrics.enabled | bool | `true` | enable system metric stream (kubelet) [ref](https://www.elastic.co/docs/current/integrations/kubernetes/kubelet#system) |
 | kubernetes.system.metrics.vars | object | `{}` | system metric stream vars |
 
-### 3 - User Extra Integrations
+### 4 - System integration
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| system.enabled | bool | `false` | enable System integration. |
```

Comment:
Why is the default `false`? I feel like most will want this on, it brings immediate value to the Observability section in Kibana.

Reply:
More than happy to have it by default as `true`, it also makes sense to me. @nimarezainia what do you think on that? 🙂